api/types/constants.go

```diff
@@ -696,6 +696,10 @@ const (
 	DiscoveryAppRewriteLabel = TeleportNamespace + "/app-rewrite"
 	// DiscoveryAppNameLabel specifies explicitly name of an app created from Kubernetes service.
 	DiscoveryAppNameLabel = TeleportNamespace + "/name"
+	// DiscoveryAppInsecureSkipVerify specifies the TLS verification enforcement for a discovered app created from Kubernetes service.
+	DiscoveryAppInsecureSkipVerify = TeleportNamespace + "/insecure-skip-verify"
+	// DiscoveryAppIgnore specifies if a Kubernetes service should be ignored by discovery service.
+	DiscoveryAppIgnore = TeleportNamespace + "/ignore"
```

Comment:
should we rename the constant to something more generic for operator usage as well? should the operator use `resources.teleport.dev/ignore` instead?

I would prefer a single annotation though.

Reply:
I used teleport.dev to keep consistency with all other labels used by app discovery :sob: